proxy/envoy/discovery.go

```diff
@@ -515,20 +528,21 @@ func (ds *DiscoveryService) ListSecret(request *restful.Request, response *restf
 	_, secret := buildIngressRoutes(ds.Mesh, ds, ds)
 
 	if secret == "" {
+		glog.V(3).Infof("Secret is not set")
```

Comment:
The Ingress handling of secret caused me a lot of pain.
We should have better logs in general.